kubemarine/procedures/check_iaas.py

```diff
@@ -272,6 +276,11 @@ def system_distributive(cluster):
             raise TestFailure("Unsupported version: %s" % ", ".join(detected_unsupported_version),
                               hint="Reinstall the OS on the host to one of the supported versions: %s" % \
                                       ", ".join(supported_versions))
+        if (result):
+            print ("Detected versions are similar")
+        else:
+            raise TestWarn ("Detected OS versions are not similar",
+            hint="Install similar verions of OS on all nodes")                                      
```

Comment:
Probably add information to the warn message, what unique pairs (OS family, OS version) are installed.